website/vmc.erb

```diff
@@ -28,12 +28,18 @@
                 <li<%= sidebar_current("docs-vmc-resource") %>>
                     <a href="#">Resources</a>
                     <ul class="nav nav-visible">
-                        <li<%= sidebar_current("docs-vmc-resource-sddc") %>>
-                            <a href="/docs/providers/vmc/r/sddc.html">vmc_sddc</a>
-                        </li>
                         <li<%= sidebar_current("docs-vmc-resource-public-ip") %>>
                             <a href="/docs/providers/vmc/r/public_ip.html">vmc_public_ip</a>
                         </li>
+                        <li<%= sidebar_current("docs-vmc-resource-sddc") %>>
+                        <a href="/docs/providers/vmc/r/sddc.html">vmc_sddc</a>
+                        </li>
```

Comment:
why is SDDC after public IP? public IP also depends on SDDC creation

Reply:
I think we should move SDDC back to where it was

Reply:
This has nothing to do with dependency. I moved it so as to have alphabetical ordering.

Reply:
Right, this is documentation, but I was thinking in terms of usability. SDDC is the first resource to create and rest follow since they depend on it. Alphabetical is ok.